src/sagemaker/remote_function/client.py

```diff
@@ -90,6 +88,8 @@ def remote(
     spark_config: SparkConfig = None,
     use_spot_instances=False,
     max_wait_time_in_seconds=None,
+    use_torchrun=False,
+    nproc_per_node=1,
```

Comment:
We need to add docstrings for these params for better user experience? Also nit: `nproc_per_node ` might not be an apt name here, wanted to check if there is value to rename it to something more informative.

Reply:
Added docstring for `use_torchrun` and `nproc_per_node` under sagemaker/remote_function/client.py